x-pack/plugins/data_enhanced/public/search/sessions_mgmt/lib/api.ts

```diff
@@ -110,6 +110,7 @@ export class SearchSessionsMgmtAPI {
         perPage: mgmtConfig.maxSessions,
         sortField: 'created',
         sortOrder: 'asc',
+        filter: nodeBuilder.is(`${SEARCH_SESSION_TYPE}.attributes.persisted`, 'true'),
```

Comment:
This isn't working properly. I think you'll need to convert this into a string instead of a `KueryNode` since it's being sent to the server.

Reply:
Fixed for now by using searchFeilds with search term.